### PR TITLE
lease: fix restore

### PR DESCRIPTION
**Pull request by xiang90**

### CODE REVIEW

File: lease/lessor.go [hunk 1]
```diff
@@ -378,9 +378,9 @@ func (le *lessor) initAndRecover() {
 		}
 		ID := LeaseID(lpb.ID)
 		le.leaseMap[ID] = &Lease{
-			ID:  ID,
-			TTL: lpb.TTL,
-
+			ID:      ID,
+			TTL:     lpb.TTL,
+			itemSet: make(map[LeaseItem]struct{}),
```
Comment (gyuho) [inline lease/lessor.go]: Shouldn't this be located under these comments?

Comment (xiang90) [inline lease/lessor.go]: sorry. you are right. fixed.

**Comment by heyitsanthony** [comment]
lgtm

**Comment by gyuho** [comment]
lgtm. Thanks!

**Pull request merged**